src/web3py/extensions/staking_vaults.py

```diff
@@ -0,0 +1,149 @@
+import logging
```

Comment:
Move some code to Vault contract and some to separate service

Reply:
done